app/views/shared/_head.html.haml

```diff
@@ -11,7 +11,7 @@
 = content_for :head
 
 = javascript_include_tag '/assets/javascripts/i18n'
-= javascript_include_tag '/assets/javascripts/i18n/' + I18n.locale.to_s
+= javascript_include_tag '/assets/javascripts/i18n/' + I18n.locale.to_s + "?v=#{i18n_md5_hash(I18n.locale.to_s)}"
```

Comment:
We use this in one other view as well, the stats embedding view that you created a while ago.

Maybe the whole thing should be turned into a helper method like `i18n_javascript_tag` so we can use the same pattern both places without duplication.